networking/k8s_nmstate/k8s-nmstate-updating-node-network-config.adoc

```diff
@@ -15,6 +15,8 @@ For more information about how to install the NMState Operator, see xref:../../n
 ====
 You cannot provide any configuration that modifies the br-ex bridge, an OVN-Kubernetes-managed Open vSwitch bridge. However, you can configure a customized br-ex bridge.
```

Comment:
I think we want to add the note about underlying interfaces, bonds, and VLANs here.